roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: setup firewall
+  import_tasks: firewall.yml
+- name: Deploy aci
+  shell: oc apply -f {{ aci_deployment_yaml_file }}
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+  run_once: True
+- name: Run kube proxy, needed for ACI
+  import_role:
+    name: kube_proxy_and_dns
+  run_once: True
+- name: include lib_openshift
+  include_role:
+    name: lib_openshift
+- name: Create namespaces
+  oc_project:
+    state: present
+    name: "{{ item }}"
+  with_items: "{{ annotate_namespace_list }}"
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+- name: Get aci policy tenant name
```

Comment:
Use `yedit`, similar to
```
  - name: Read master storage backend setting
    yedit:
      state: list
      src: /etc/origin/master/master-config.yaml
      key: kubernetesMasterConfig.apiServerArguments.storage-backend
    register: _storage_backend
```

Reply:
Hi @vrutkovs thank you for review. Does yedit works with yaml having multiple document ?

Reply:
Hi @vrutkovs , I tried yedit, but look to me it doesn't support reading from yaml file which has multiple document. We are trying to read the value of key from a deployment file which has multiple documents.

Reply:
Ah, I see. Okay, lets use shell here then